### DISCUSSION TITLE
Hide selected in MultiValue

### DISCUSSION THREAD

**Discussion by maxizhukov** [JedWatson/react-select#4533] [Q&A]
Hello is here some way, how I can I can another badge in MultiValue if more then 2 values are selected? Like if here is more then 2 are already selected not to show every one and show only "4 items are selected"?
![Screenshot 2021-03-29 at 09 49 43](https://user-images.githubusercontent.com/63158421/112803992-2d90c300-9074-11eb-9edd-605c4758f14f.png)

**Comment by ebonow** [comment]
Greetings @maxizhukov ,

Perhaps the solution presented in https://github.com/JedWatson/react-select/issues/3856 can be helpful.

Additionally, this was recently asked in another discussion in which an issue was found with the above approach regarding menu closing behavior, but it might still give some direction on a correct implementation.
https://github.com/JedWatson/react-select/discussions/4500
